packages/cli/src/util/dev/server.ts

```diff
@@ -544,6 +544,29 @@ export default class DevServer {
     return undefined;
   }
 
+  exposePrefixedEnvVars({
+    envPrefix,
+    envs,
+  }: {
+    envPrefix: string | undefined;
+    envs: Env;
+  }): Env {
+    // TODO: expose VERCEL_GIT_ env vars as well once we add support (VCCLI-303)
+    const allowed = ['VERCEL_URL', 'VERCEL_ENV'];
+    const newEnvs: Env = {};
```

Comment:
This looks copy/pasta from `@vercel/build-utils`. Can we reuse the existing function?

Reply:
Thanks! updated